airflow/providers/elasticsearch/hooks/elasticsearch.py

```diff
@@ -0,0 +1,89 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from es.elastic.api import connect
+
+from airflow.hooks.dbapi_hook import DbApiHook
+
+
+class ElasticsearchHook(DbApiHook):
+    """
+        Interact with Elasticsearch through the elasticsearch-dbapi
+    """
+
+    conn_name_attr = 'elasticsearch_conn_id'
+    default_conn_name = 'elasticsearch_default'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.schema = kwargs.pop("schema", None)
+        self.connection = kwargs.pop("connection", None)
+
+    def get_conn(self):
+        """
+        Returns a elasticsearch connection object
+        """
+        conn_id = getattr(self, self.conn_name_attr)
```

Comment:
Is it necessary?  Won't `conn_id = self.conn_name_attr` work?

Reply:
Ah yes, it was necessary on the `pylint`